packages/api-gateway/__tests__/src/drivers/http/routes/chats/get-chat-messages.test.js

```diff
@@ -1,36 +1,41 @@
 // External dependencies
 const { faker } = require('@faker-js/faker');
-const { listChatMessages } = require('@booking-services/messages');
+const { listChatMessages, isChatRelatedToUser } = require('@booking-services/messages');
 
 // Internal dependencies
 const { fastify } = require('../../../../../../src/drivers/http/server');
 
 // Mocks
-const customerID = faker.datatype.uuid();
-const hostID = faker.datatype.uuid();
-
-const mockListChatMessages = [
-  {
-    _id: faker.datatype.uuid(),
-    bookingId: faker.datatype.uuid(),
-    hostId: hostID,
-    customerId: customerID,
-    createdAt: faker.datatype.datetime(),
-    updatedAt: faker.datatype.datetime(),
-    deletedAt: null,
-  },
-];
+const userID = faker.datatype.uuid();
+
+const mockListChatMessages = {
+  pages: 1,
+  messages: [
+    {
+      _id: faker.datatype.uuid(),
+      chatId: faker.datatype.uuid(),
+      text: faker.lorem.sentence(),
+      createdBy: userID,
+      createdAt: faker.datatype.datetime(),
+      deletedAt: null,
+    },
+  ],
+};
 
 describe('GET /chats/{chatId}/messages', () => {
   describe('given an authenticated user and a valid chatId', () => {
     // TODO: Change this once we have authentication ready
-    const bearerToken = customerID;
+    const bearerToken = userID;
```

Comment:
I think it would be better rename this var to host

Reply:
I thought the same thing, but since this endpoint can be used by both customers and hosts, I decided to use `user` to be a bit more generic.